components/automate-ui/src/app/app-routing.module.ts

```diff
@@ -176,14 +175,6 @@ const routes: Routes = [
               path: '',
               component: NotificationsComponent
             },
-            {
-              path: 'form',
-              component: NotificationFormComponent
-            },
-            {
-              path: 'form/:id',
-              component: NotificationFormComponent
-            },
```

Comment:
Why are these no longer needed?

Reply:
yes, we don't need this, because now we can create notification using the modal pop and for edit and show our consistency changes are already merged.